content/techniques/sql.md

```diff
@@ -743,6 +743,28 @@ TypeOrmModule.forRootAsync({
 
 This construction works the same as `useClass` with one critical difference - `TypeOrmModule` will lookup imported modules to reuse an existing `ConfigService` instead of instantiating a new one.
 
+If you don't use ConnectionName as 'default', you need to specify ConnectionName so that nestjs can recognize it.
+
+```typescript
+TypeOrmModule.forRootAsync({
+  useFactory: () => {
+    return {
+      type: 'mysql',
+      host: 'localhost',
+      port: 3306,
+      username: 'root',
+      password: 'root',
+      database: 'test',
+      entities:[],
+      synchronize: true,
+      name:"typeorm_user_connection"
+    }
+  },
+  name: 'USER',
+}),
+```
```

Comment:
```suggestion
> **Hint**: make sure the `name` property is on the same level as the `useFactory`, `useClass`, or `useValue` property. This will allow Nest to properly create the injection token
```

This will help allow cut down on code, and still illustrate the point being made, in my opinion

Reply:
ok, i am satisfied.